tarantool/connection.py

```diff
@@ -298,6 +329,52 @@ def connect_unix(self):
             self.connected = False
             raise NetworkError(e)
 
+    def wrap_socket_ssl(self):
+        '''
+        Wrap an existing socket with SSL socket.
+
+        :raise: NetworkError
+        :raise: `ssl.SSLError`
+        '''
+        if not is_ssl_supported:
+            raise NetworkError("SSL is unsupported by the python.")
+
+        ver = sys.version_info
+        if ver[0] < 3 or (ver[0] == 3 and ver[1] < 5):
+            raise NetworkError("SSL transport is supported only since " +
```

Comment:
No it is not the reason.

I think that for new features, we can limit to a less supported version. It's 3.7 now:
https://devguide.python.org/#status-of-python-branches

But I choose 3.5 because we test this version on CI. Considering that in `ssl` module many changes depend on version it may be difficult to support lower versions.